src/views/text_view.rs

```diff
@@ -146,6 +171,18 @@ impl TextContentInner {
             Some(ref last) => last.x.accept(size.x) && last.y.accept(size.y),
         }
     }
+
+    fn get_value(&self) -> Arc<StyledString> {
+        Arc::clone(&*self.content_value.borrow())
+    }
+
+    fn get_cache(&self) -> Arc<StyledString> {
```

Comment:
Here too let's return `&mut Arc` - most of the time we won't even need to clone the `Arc`.